package.json

```diff
@@ -38,7 +38,10 @@
     "node": ">=10.12"
   },
   "peerDependencies": {
-    "cdktf": "^0.2.2",
+    "@cdktf/provider-aws": "^1.0.122",
```

Comment:
This is a peer dependency so that our downstream use cases can also pull in the providers